<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="ipWid" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>561</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="tftpWid" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>571</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="ipsTabWid" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>300</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="oksTabWid" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>300</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="errsTabWid" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>300</y>
      <width>300</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1216</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
